client/main.lua

```diff
@@ -370,32 +334,99 @@ CreateThread(function()
                         end
                     end
                 end
+                Wait(100)
+            else
+                Wait(1000)
             end
         end
-
-        Wait(sleep)
-    end
-end)
+    end)
+end
 
 -----------------------
 ---- Client Events ----
 -----------------------
+local togglelocksBind
+togglelocksBind = lib.addKeybind({
+    name = 'togglelocks',
+    description = locale('info.toggle_locks'),
+    defaultKey = 'L',
+    onPressed = function()
+        togglelocksBind:disable(true)
+        setVehicleDoorLock(getVehicle(), nil, true)
+        Wait(1000)
+        togglelocksBind:disable(false)
+    end
+})
+
+local engineBind
+engineBind = lib.addKeybind({
+    name = 'engine',
+    description = locale('info.engine'),
+    defaultKey = 'G',
+    onPressed = function()
+        engineBind:disable(true)
+        toggleEngine()
+        Wait(1000)
+        engineBind:disable(false)
+    end
+})
 
-RegisterKeyMapping('togglelocks', locale('info.toggle_locks'), 'keyboard', 'L')
-RegisterCommand('togglelocks', function()
-    setVehicleDoorLock(getVehicle(), nil, true)
-end, false)
+AddEventHandler('ox_lib:cache:vehicle', function()
+    showHotwiringLabel()
+end)
+showHotwiringLabel()
+
+RegisterNetEvent('QBCore:Client:VehicleInfo', function(data)
```

Comment:
What is this event supposed to do? Is this supposed to be a function in lib.onCache?

Reply:
qbx_core/bridge/qb/server/events.lua 2th line

Reply:
this can be rewritten to baseevents:enteringVehicle

Reply:
Okay I didn't know that was an event lol.